docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,110 @@
+    ```
+    ├── organizations
+      └── peerOrganizations
+          └── org1.example.com
+              ├── msp
+              └── peers
+                  └── peer0.org1.example.com
+                      ├── msp
+                      └── tls
+    ```
+2. While **cryptogen** is a convenient utility that can be used to generate certificates for a test network, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you. Regardless of which certificate provider you choose, you should have already used it to generate the peer enrollment certificate, TLS certificate, private keys, and the MSPs that Fabric must consume. Refer to the [CA deployment](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html) and [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topics for instructions on how to generate these certificates. You need to generate the following sets of certificates:
+    - **Peer TLS CA certificates**
+    - **Peer organization certificates (organization MSP)**
+    - **Peer enrollment certificates (peer local MSP)**  
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, most users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **TLS certificates**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `peer.tls.rootcert.file` parameter in `core.yaml`.
+        - After you have generated the peer TLS certificate, the certificate will have been generated in the `signcerts` directory, and the private key will have been generated in the `keystore` directory. Rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
+        - Copy the peer TLS certificate and private key to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls`. The path and name of the certificate and private key files correspond to the `peer.tls.cert.file` and `peer.tls.key.file` parameters in the `core.yaml`.
+      - **Peer organization MSP**:
```

Comment:
I'm thinking this whole bullet and sub-bullets can be removed now.
We're not configuring the org MSP here, just the peer's local MSP.  The org MSP would be configured when creating the channel.
Or am I missing the intent?

Reply:
I agree. I think Pam added this because it was part of her process for deploying the peer (since it is a prerequisite). But it's covered in the CA docs so I think we can probably scrap this little section.

What do you think @pamandrejko?

Reply:
I checked the [Fabric CA deployment guide](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#folder-structure-for-your-org-and-node-admin-identities) and this specific information not in there. It just says to create the `tls` folder in the folder structure. I think the intent was to fill in the TLS  certs later when you create the node. While you use the CA to register and enroll the peer identity, when you deploy the peer, you have to build out this folder structure, and because this is a tutorial I am providing the specific instructions how to do this. 

So I suppose we have a choice then, include it in here, or add more details to the CA deployment guide. 

I am suggesting we do it here (but could be persuaded otherwise) that before you create the peer you create the organization msp. It feels disjoint to me to tell them to insert the peer org mspid into the core.yaml even though the msp does not already exist. This is an error prone step that will tank your peer if not done correctly, so I thought it was worth explicitly including it here.

Reply:
I agree that the org MSP should be created prior to deploying a peer.
However, that should not be WITHIN these instructions, it should be done BEFORE these instructions.
It is in fact described fairly well in the CA deployment guide here: https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#create-the-org-msp-needed-to-add-an-org-to-a-channel
So perhaps we just need to link to that at the BEGINNING of this topic, not WITHIN this topic.

Reply:
OK. That does make sense. Partnered with you other suggestion of creating the 3 sections in the pre-reqs (Certs/storage/cfg), it comes together nicely then. 

I would suggest however that we move these specific instructions into the Fabric CA doc (I am happy to do that).